deployment/roles/common/tasks/logging.yml

```diff
@@ -0,0 +1,29 @@
+- name: Set the local docker logs configuration file
+  template:
+    src: 30-docker.conf.j2
+    dest: /etc/rsyslog.d/30-docker.conf
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Set the log configuration file to send docker logs to remote server
+  template:
+    src: 35-docker-remote-logging.conf.j2
+    dest: /etc/rsyslog.d/35-docker-remote-logging.conf
+    owner: root
+    group: root
+    mode: 0644
+  when: syslog_server_port is defined
```

Comment:
I think that commented out `syslog_server_port` option should also be added to `hosts.yml.example` in `ui` section as well

Reply:
Done.